proposals/schema.graphql

```diff
@@ -12,6 +8,9 @@ type Proposal @entity {
   module: ProposalModule!
   num: Int!
   open: Boolean!
+  createdAt: Date! @index
+  executedAt: Date @index
+  closedAt: Date @index
```

Comment:
are these semantically different or could we just call this `completedDate`?

Reply:
this difference gives us information about why it became no longer open, think it might be worth keeping